src/preview/utils.ts

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto';
```

Comment:
My suggestion with sha256 was to avoid adding config var, if we add a config var we could simply add SECRET=blabla and just check if token === SECRET, not see why we would need 2 config vars here

Reply:
All config vars removed, and sha256 encrypted string hardcoded